Maxfurry/Store-Manager
# [Feature #161204951] eslint config and errors [ESLINT]

#### What does this PR do?
Makes changes to the eslint rule from 4 spaces indentation to that of 2

#### Description of Task to be completed?
Have the following files follow the rules and are error free 

/src/controller/product.js
/src/controller/sales.js
/src/controller/users.js

/src.route/product.js
/src.route/sales.js
/src.route/users.js

/server.js
                    
#### How should this be manually tested?
After cloning the repository
run npm install
When installations are complete
Open the project files one by one
Eslint should not prompt for any error

#### Any background context you want to provide?
Previously it had an indentation of 4 spaces which is quite large 
so a 2 space indentation was requested

#### What are the relevant pivotal tracker stories?
#161204951
